### ISSUE TITLE
Missing forms and megas - Gallade, Diancie, Hoopa, etc

### ISSUE THREAD

**Issue by sattmaunders** [veekun/pokedex#141]
Is anyone working on updating the database, specifically to be current for ORAS?
Anyone have an updated version out there, or an alternative?

**Comment by magical** [comment]
Nobody's working on it, as far as i know. We've all been busy with other things. Sorry :(

**Comment by mschex1** [comment]
I am using it and will try to add them for you.

**Comment by sattmaunders** [comment]
Fair enough - when I get finished with my current project I'll take a stab at adding some of the info. 

@magical No worries :)! 
@mschex1 Awesome :D!

**Comment by StAlRuth** [comment]
I have all of the OR/AS data in JSON format over at [StAlRuth/Gen6](http://github.com/StAlRuth/Gen6), in case it helps.

**Comment by eevee** [comment]
I am, in fact, actually, gradually, looking into this.

**Comment by eevee** [comment]
From my notes, these things remain:

- [ ] held items
- [ ] localized form names and the like
- [ ] meta stuff for new moves
- [ ] write new move effects
- [ ] write effect text for all the "unknown" xy moves
- [ ] encounters (!)
- [ ] item sprites maybe?
- [ ] new pokémon dex sprites, which should fill in a lot of blank megas
    - difficulty: there are no cosplay pikachu dex sprites.  or, rather, they are, but they're just blank images.
- [ ] menu sprites

I'm working on sprites at the moment; only real remaining issue is figuring out filenames for forms, which is slightly tedious.

**Comment by StAlRuth** [comment]
If you need the minisprites I have them all [here](http://static.tumblr.com/jjqlt6d/jVbnwjtru/oras_pokemon.png)

**Comment by matthewdias** [comment]
any idea on when oras data will be available in a database format? If anything needs done I'm willing to help out but its kind of hard to follow what is completed and what isn't.

**Comment by eevee** [comment]
The stuff that's live on the site now is already in the database, in this repo.

**Comment by matthewdias** [comment]
oh cool. is there anything missing from the site that i could help out with? or is the data complete?

**Comment by eevee** [comment]
There's the checklist from my earlier comment...  :)

**Comment by zaneadix** [comment]
Hey guys, just poking in (from pokeapi) to see if theres been any movement on OR/AS data. We've got some interest right now in our slack and theres some discussion about some of us possibly trying to contribute.

**Comment by eevee** [comment]
I'm probably going to devote some time this month to veekun stuff.  Is there something in particular you need most urgently?

**Comment by craig-clayton** [comment]
@eevee I believe the biggest thing right now would likely be location data for Pokemon encounters. If we can help out with anything, please do let us know. :)

**Comment by metaunicorn** [comment]
I could also contribute, maybe we can organize/distribute the effort a bit.
Have you seen the new "Projects" feature in Github? We can use that https://github.com/veekun/pokedex/projects

**Comment by mikazo** [comment]
Hey, I'm also willing to contribute to adding ORAS encounter data. Has there been an organized effort? Just let me know and I'll help out.

**Comment by Serenitor** [comment]
> From my notes, these things remain:
> 
> * [ ]  held items
> * [ ]  localized form names and the like
> * [ ]  meta stuff for new moves
> * [x]  write new move effects
> * [x]  write effect text for all the "unknown" xy moves
> * [ ]  encounters (!)
> * [ ]  item sprites maybe?
> * [x]  new pokémon dex sprites, which should fill in a lot of blank megas
>   
>   * difficulty: there are no cosplay pikachu dex sprites.  or, rather, they are, but they're just blank images.
> * [x]  menu sprites
> 
> I'm working on sprites at the moment; only real remaining issue is figuring out filenames for forms, which is slightly tedious.

is there any glimmer of hope that this checklist is going to get some attention in the foreseeable future? for me, **the missing encounter data, particularly for ORAS and USUM, would be crucial**.
right now, I have it scraped from various pokedex websites (mainly bulbapedia), but the data is rather inconsistent at times, and getting all the details directly from the game data would be really amazing...

**Comment by magical** [comment]
I will have some time this summer. I should be able to work on ripping the encounter data.
